modules/tensor_mechanics/include/materials/ComputeDilatationThermalExpansionFunctionEigenstrain.h

```diff
@@ -27,18 +27,10 @@ class ComputeDilatationThermalExpansionFunctionEigenstrainTempl
 protected:
   /*
    * Compute the fractional linear dilatation due to thermal expansion delta L / L
```

Comment:
Doxygen comment should reflect the templated nature of this method (i.e. does different things for different cases)